.github/workflows/label-slow-prs.yml

```diff
@@ -0,0 +1,13 @@
+name: Label slow PRs
+
+on:
+  schedule:
+    - cron: "0 2 * * MON"
```

Comment:
Why not running it every day?

Reply:
Could be a bit wasteful if we're only looking at the results on a weekly basis...

Reply:
ok